.github/workflows/ci.yml

```diff
@@ -65,3 +65,15 @@ jobs:
           -Psigning.keyId="${{ secrets.SIGNING_KEY_ID }}"
           -Psigning.password="${{ secrets.SIGNING_PASSWORD }}"
           -Psigning.secretKeyRingFile="$HOME/secring.gpg"
+
+      - name: PublishDocumentation
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          ./gradlew dokkaHtml
+          git config user.email "ci-build@juul.com"
+          git config user.name "ci-build"
```

Comment:
Git required I specify some credentials to push up. Do we have a config somewhere I didn't see?

Reply:
We do not, though we can setup any variables that you need on the repo.
Will you also need to configure SSH keys for the push?
(I don't know what the standard/typical setup is for having to push from a GitHub Action)

Is there perhaps a pre-made GitHub Action that might make this easier?

Reply:
I think I may have misunderstood at first...
Basically, it is already working, you just need to know what use the commit will appear under?

Reply:
The following GitHub Secrets have been configured:

- `CONX_GH_PAGES_USER_EMAIL`
- `CONX_GH_PAGES_USER_NAME`

Reply:
Yeah, the push works just fine as is.